integration

```diff
@@ -993,6 +993,7 @@
   "yinyang17/pvpc_energy",
   "yo-han/Home-Assistant-Carelink",
   "YorkshireIoT/ha-google-fit",
+  "cdnninja/yoto_ha",
```

Comment:
This list need to be sorted alphabetically.

Reply:
Fixed.